server/config.ts

```diff
@@ -63,6 +63,14 @@ export default {
       },
       agent: new AgentConfig(),
     },
+    assessRisksAndNeedsApi: {
+      url: get('ASSESS_RISKS_AND_NEEDS_API_URL', 'http://localhost:8094', requiredInProduction),
+      timeout: {
+        response: Number(get('ASSESS_RISKS_AND_NEEDS_API_TIMEOUT_RESPONSE', 10000)),
+        deadline: Number(get('ASSESS_RISKS_AND_NEEDS_API_TIMEOUT_DEADLINE', 10000)),
```

Comment:
given we saw a 8-9 second referral post to ARN, can we increase this to 20s, please?